src/server/services/wms/qgswmsrenderer.cpp

```diff
@@ -68,6 +68,8 @@
 #include "qgsdxfexport.h"
 #include "qgssymbollayerutils.h"
 #include "qgsserverexception.h"
+#include "qgsexpressioncontextutils.h"
```

Comment:
Remove this line in cc5d449

Reply:
I don't think that import is really necessary, I might have copy pasted it by mistake.